cnnlabs/cnn-antools-factors-fe
# CNNETANTWO- 61 Subtask of CNNETANTWO- 57 - Alert Module

This is for subtask Alert Module:http://tickets.turner.com/browse/CNNETANTWO-61

steps to test:

1. pull down this branch.
2. start up the app
3. goto http://localhost:5050/content-details/http%3A%2F%2Fmoney.cnn.com%2F2017%2F07%2F06%2Fnews%2Feconomy%2Frussia-china-investment-deal-sanctions%2Findex.html
4. make a change to the headline and click cancel
5. you should see the alert dialog box to confirm your cancel request with a warning about unsaved changes.
6. leave should take to you back to content list page
7. stay should close dialog box with changes still intact on the content detail page


SCREEN SHOT
![image](https://user-images.githubusercontent.com/3534608/28475343-b8abe9fa-6e19-11e7-87ed-1ed177da20eb.png)
